Packs/CiscoASA/ReleaseNotes/1_0_33.md

```diff
@@ -0,0 +1,4 @@
+
+#### Modeling Rules
+##### Cisco ASA Modeling Rule
+- update Modeling Rule
```

Comment:
@guytamir10 Please elaborate more on the fix, something like "added user extraction to event id ...."